components/rendering/src/markdown.rs

```diff
@@ -62,6 +70,11 @@ fn find_anchor(anchors: &[String], name: String, level: u16) -> String {
     find_anchor(anchors, name, level + 1)
 }
 
+/// Colocated asset links refers to the files in the same directory.
+fn is_colocated_asset_link(link: &str) -> bool {
+    !link.starts_with('/') && !link.starts_with('#') && !STARTS_WITH_SCHEMA_RE.is_match(link)
```

Comment:
Does it make sense to consider `../` as well? Those assets also wouldn’t be colocated, but I don’t know if those links could occur while using zola in a reasonable way.

Reply:
For consistency we want to change links starting with `../` as well. Imagine you want to link to `section/asset.png` from `section/page/index.md`, then you could write `../asset.png`. Without link rewriting the link would be incorrect then.

Reply:
Okay, makes sense. What exactly is `'/'` for then?

Reply:
`/` is for absolute links. We should probably prepend `base_url`, but not the page permalink.

Reply:
Okay, thanks for explaining! :)